platform-validation/scripts/aziot-compatibility.sh

```diff
@@ -553,6 +553,42 @@ check_shared_library_dependency_display_util() {
     esac
 }
 
+check_package_manager() {
+    not_found=0
+    result="$(need_cmd apt-get)"
+    if [ $? != 0 ] ; then
```

Comment:
Thinking a little more about this, I think we should have 2 checks 

check_supported_package_manager -> checks for apt-get -> We should not even have a warning for this because they may have dpkg/rpm support. 
Based on the above check -> do the following

if there is apt-get support -> Skip the below checks
check_supported_package_type -> Checks for dpkg, rpm
check_additional_packages -> check for ca_certificate 

What do you think about this?

Reply:
I think this is good flow. i will change them and add it

Reply:
Also, you can probably do a for loop and break after the first match instead of a nested if ?